README.md

```diff
@@ -28,12 +28,48 @@ When the reader has completed this Code Pattern, they will understand how to:
 * [Flask](http://flask.pocoo.org/): is a micro web framework.
 * [ngrok](http://ngrok.com/): allows you to open an HTTPS tunnel into your REST API.
 
-## Prerequisites:
+## Featured technologies
+
+* [Artificial Intelligence](https://medium.com/ibm-data-science-experience): Artificial intelligence can be applied to disparate solution spaces to deliver disruptive technologies.
+* [Python](https://www.python.org/): Python is a programming language that lets you work more quickly and integrate your systems more effectively.
+
+## Prerequisites
 
 The following are prerequisites to start developing this application:
 
 * Xcode
 * Turi Create (5.0 Beta 2)
 * Cocoapods
 * Python 2.7 & PIP
+* Flask
 * Jupyter
+* ngrok
+
+# Steps
+
+(NOTE: I still need to add screenshots/pictures to all the steps!)
```

Comment:
If this was a `TODO`, then it'd be easier to find and remove later.. but I think it could also go without saying.

Reply:
and we should provide install links for each of the prereq's